### PR TITLE
log4j and UTF and rename var

### PR DESCRIPTION
**Pull request by sumitnigamcs**
dependency log4j 1.2.17 changed to log4j api 2.16
UTF-8 encoding added
renamed a variable suggested in #814

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java [hunk 1]
```diff
@@ -28,7 +29,7 @@
  */
 @Deprecated
 public class BigQueryActions {
-  private static final Logger logger = Logger.getLogger(BigQueryActions.class);
+  private static final Logger logger = LogManager.getLogger(BigQueryActions.class);
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java]: why do we need this change? Isn't bumping up log4j version sufficient?

File: pom.xml [hunk 2]
```diff
@@ -19,7 +19,10 @@ the License.-->
   <groupId>io.cdap.tests.e2e</groupId>
   <artifactId>cdap-e2e-framework</artifactId>
   <version>0.0.1-SNAPSHOT</version>
-
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```
Comment (rmstar) [inline pom.xml:24]: why do we need to add these properties?

File: src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java (hunk 1 above)
Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java]: instead of using log4j 1.2.x, now log4j api has been used for moving it to 2.x version. For that, classes has been modified

File: pom.xml (hunk 2 above)
Comment (sumitnigamcs) [reply pom.xml:24]: a warning was shown while building the project, that encoding is not defined, and we were giving it manually on our VMs, else it used to give errors. So added the same as properties

File: src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java (hunk 1 above)
Comment (tivv) [reply src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java]: Please move to slf4j / logback as this is logging solution that CDAP uses

**Comment by rmstar** [comment]
There's a test failure
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:testCompile (default-testCompile) on project google-cloud: Compilation failure
[ERROR] /tmp/cdap-e2e-tests/cdap-e2e-tests/plugin/src/e2e-test/java/io/cdap/plugin/gcp/stepsdesign/GCSBasicDemo.java:[162,56] cannot find symbol
[ERROR]   symbol:   variable myObj
[ERROR]   location: class stepsdesign.BeforeActions

```

**Comment by sumitnigamcs** [comment]
> There's a test failure
> 
> ```
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:testCompile (default-testCompile) on project google-cloud: Compilation failure
> [ERROR] /tmp/cdap-e2e-tests/cdap-e2e-tests/plugin/src/e2e-test/java/io/cdap/plugin/gcp/stepsdesign/GCSBasicDemo.java:[162,56] cannot find symbol
> [ERROR]   symbol:   variable myObj
> [ERROR]   location: class stepsdesign.BeforeActions
> ```

yeah, i have raised another PR for the same : https://github.com/data-integrations/google-cloud/pull/861

**Review by rmstar — APPROVED** [review]

**Review by rmstar — CHANGES_REQUESTED** [review]
sorry, approved this by mistake. please address logging comment

File: src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java (hunk 1 above)
Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java]: Working on it. 
Somehow even after changing implementation to logback in e2e-framework, google-cloud module is expecting log4j.properties, logback.xml is not working. Looking into the issue

Comment (tivv) [reply src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java]: You may need to include https://logging.apache.org/log4j/2.x/log4j-to-slf4j/index.html instead of log4j. Check google-cloud project, it should be solving a similar problem

Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java]: added log4j12 exclusions in hbase dependencies under main dependencies section to make logback work, please confirm if that is ok

Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/BigQueryActions.java]: @tivv In google-cloud project, i have added exclusions for lo4j12 in hbase dependencies under main dependency section, to make logback work in that project. Please confirm if this change is acceptable. Those changes can be seen in https://github.com/data-integrations/google-cloud/pull/814

File: src/main/java/io/cdap/e2e/utils/SeleniumDriver.java [hunk 3]
```diff
@@ -77,7 +78,7 @@ public static void getWaitDriver() {
 
   public static void openPage(String url) {
     logger.info(url);
-    logger.info(chromeDriver);
+    logger.info(String.valueOf(chromeDriver));
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/utils/SeleniumDriver.java:81]: why is this change needed?

Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/utils/SeleniumDriver.java:81]: log4j's logger.info was accepting object as param, whereas logback is not

**Review by rmstar — APPROVED** [review]
please squash commits

**Comment by sumitnigamcs** [comment]
> please squash commits

done @rmstar
